supervisor/workflow.h

```diff
@@ -31,6 +31,6 @@ void supervisor_workflow_reset(void);
 // True when the user could be actively iterating on their code.
 bool supervisor_workflow_active(void);
 
-void supervisor_workflow_request_background(void);
+void supervisor_workflow_request_background(void *data);
```

Comment:
Why add this argument? I don't see it used anywhere.

Reply:
At one point I was passing some context between threads. I left it there rather than using a needless function call just to strip the data arg. I'll clean it up a bit.